attachment_synchronize/models/task.py

```diff
@@ -112,10 +112,10 @@ def _template_render(self, template, record):
         return render_result
 
     @api.model
-    def run_task_scheduler(self, domain=None):
-        if domain is None:
-            domain = []
-        domain.append([('method_type', '=', 'import')])
+    def run_task_scheduler(self, domain=list()):
+        if ('method_type', '=', 'import') not in domain:
+            domain.append([('method_type', '=', 'import')])
+
```

Comment:
@florian-dacosta i agree, i've changed mainly because domain is passed from cron so no need to add again and not sure if method is called from elsewhere, such as manually without domain.